src/dds/rs-dds-device-proxy.cpp

```diff
@@ -209,6 +209,28 @@ dds_device_proxy::dds_device_proxy( std::shared_ptr< context > ctx, std::shared_
             }
         } );  // End foreach_stream lambda
 
+    for( auto & sensor_info : sensor_name_to_info )
+    {
+        sensor_info.second.proxy->initialization_done( dev->device_info().product_id, dev->device_info().product_line );
```

Comment:
I expect both arguments to the function will go away: note that, even now, both are optional. In fact, I think we can consider removing `product_line`, as well...